thebattleforwesnoth/egg-battle-for-wesnoth.json

```diff
@@ -0,0 +1,62 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-04-20T10:04:36-04:00",
+    "name": "The Battle for Wesnoth",
+    "author": "mattman107@github.com",
+    "description": "Egg for hosting a Battle for Wesnoth Server.",
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/parkervcp\/games:thebattleforwesnoth": "ghcr.io\/parkervcp\/games:thebattleforwesnoth"
+    },
+    "file_denylist": [],
+    "startup": "echo -e \"passwd=\\\"{{PASSWD}}\\\"\" > wesnothd.cfg; echo -e \"motd=\\\"{{MOTD}}\\\"\" >> wesnothd.cfg; .\/battleForWesnoth\/build\/wesnothd -p {{SERVER_PORT}} -c \/home\/container\/wesnothd.cfg",
```

Comment:
this egg things hsould be done with the parser and no directly echo to the file.

Reply:
As far as I understand from the [documentation](https://pterodactyl.io/community/config/eggs/creating_a_custom_egg.html#configuration-files) and the [source code](https://github.com/pterodactyl/wings/blob/develop/parser/parser.go#L25-L30),  '.cfg' isn't a valid file parsing option. Do one of the other options also work for '.cfg' files?

Reply:
use the file one, it is made to replace whole lines in files.

Reply:
Updated to use the file config parser.